### DISCUSSION TITLE
NuGet packages update ?

### DISCUSSION THREAD

**Discussion by Felx-B** [microsoft/Power-Fx#186] [Q&A] [answered]
Hi 👋
It's been few months since the release of NuGet packages `0.2.1-preview`
Any plan to release another (pre)version ?
Thank you  !

**Comment by MikeStall** [comment, marked as answer]
We do produce a nuget _per commit_. But we've had some breaking changes going through and haven't fully stabilized the public surface.  

They're currently published to this feed, but this may change soon:
https://dev.azure.com/ConversationalAI/BotFramework/_packaging?_a=package&feed=SDK%40Local&package=Microsoft.PowerFx.Core&protocolType=NuGet&view=overview 

You can add that feed as .nuget , like this:
https://github.com/microsoft/power-fx-host-samples/blob/b4218025b7d9781124644ed9e82c41f767ef2361/Samples/ConsoleREPL/nuget.config

**Reply by Felx-B** [reply]
Oh nice !
Thank you, I'll try this
